tutorials/clustering.md

```diff
@@ -13,17 +12,25 @@ In this tutorial, we will show how you can create a distributed hypertable withi
 a distributed database. We will set up the distributed database with one primary access node
 and two data nodes across three distinct TimescaleDB instances.
 
-Prerequisites:
-- 3 instances of TimescaleDB installed (private BETA branch only)
-- A superuser set up on all 3 instances of TimescaleDB
-- Network connectivity between the 3 TimescaleDB instances
-- Must be on PG11
+It is assumed before starting this tutorial you will have downloaded and installed the
+TimescaleDB multinode beta package on three networked machines running Postgres 11. It's
```

Comment:
FWIW, I'd like to move away from making "multinode" sound as something different. Let's just explicitly state the prerequisites here.

For instance:

```
The prerequisites for this tutorial are:

- TimescaleDB version 2.0 (beta) or greater.
- PostgreSQL 11
- Three networked PostgreSQL instances with a common superuser role
```

Reply:
I would also suggest giving these three instances (domain) names and use that in the description below so that you instead of saying "connect to the instance that you want to be an access node", you can write "connect to access.example.com".

Reply:
Name the superuser role here as well---that is, 'postgres'---and use it in the description below.

Reply:
I've incorporated all of these comments.  I actually lifted the term "TimescaleDB multinode beta" from the instructions Jeff put together, though it might be a bit late to change that now.